addons/mail/static/src/components/attachment_link_preview/attachment_link_preview.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.AttachmentLinkPreview" owl="1">
+        <t t-if="attachmentLinkPreviewView">
+            <div class="o_AttachmentLinkPreview rounded px-3 pt-3 pb-2 d-flex" t-attf-class="{{ className }}" t-ref="root">
+                <t t-if="attachmentLinkPreviewView and attachmentLinkPreviewView.attachment">
```

Comment:
The short-circuit is not necessary since `attachmentLinkPreviewView` is guarded above.
```suggestion
                <t t-if="attachmentLinkPreviewView.attachment">
```